x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/providers.tsx

```diff
@@ -111,7 +111,9 @@ const ParensContainer = styled(EuiFlexItem)`
 `;
 
 const getDataProviderValue = (dataProvider: DataProvidersAnd) =>
```

Comment:
consider removing this conditional assignment (similar to https://github.com/elastic/kibana/pull/142436#discussion_r986208635)

Reply:
fixed